truss/templates/server.Dockerfile.jinja

```diff
@@ -21,6 +21,9 @@ COPY ./base_server_requirements.txt base_server_requirements.txt
 RUN pip install -r base_server_requirements.txt --no-cache-dir && rm -rf /root/.cache/pip
 
 {% if live_reload %}
+{% if config.base_image %}
+RUN ln -sf {{config.base_image.python_executable_path}} /usr/local/bin/python
```

Comment:
It's ok to override the link here. But it would be good to print a warning if the link already exists and is being overwritten.